electrum_nmc/electrum/gui/qt/configure_dns_dialog.py

```diff
@@ -203,6 +205,10 @@ def create_address_record(self):
         if address_type == "zeronet" and self.has_zeronet_record(domain):
             self.show_error(domain + _(" already has a ZeroNet record."))
             return
+        
+        if (address_type == "ipfs" or address_type == "ipns") and self.has_dnslink_record(domain):
+            self.show_error(domain + _(" already has a IPFS or IPNS record."))
+            return
```

Comment:
Can you move this above the `zeronet` check, for consistency? Also there's a typo, `a IPFS` should be `an IPFS`.